### DISCUSSION TITLE
NFS over TCP not available & dhclient.conf nonexist

### DISCUSSION THREAD

**Discussion by keerthi-varuman** [ltsp/ltsp#680] [General] [answered]
I have installed the ubuntu 20.04 LTSP server. when i boot the LTSP client I get DHCP and kernel after that I get error that **dhclient.conf nonexist and NFS over TCP not available** screenshot attached. I have checked my network configuration and everything all looks good. i have no idea what it could be.

![IMG_20220308_211255](https://user-images.githubusercontent.com/75841376/157274540-78355cfc-7ee0-4bd0-bb7e-27af1ee74420.jpg)

**Comment by alkisg** [comment]
To be clear, you have installed the Ubuntu 20.04 server image and you're using chrootless LTSP (ltsp image /) to boot LTSP "Ubuntu server clients", right?

If so, this will need some troubleshooting; please come to the [chat room](https://ltsp.org/advanced/chat-room/) for remote support. Do arrange for screen sharing with a VM client; e.g. we can use VNC or Anydesk to any PC that has VirtualBox installed, so that we can run a Virtual LTSP client inside VirtualBox.

**Comment by intelliant01** [comment]
@keerthi-varuman The photo shared has two IP addresses 192.168.5.1 and 192.168.67.1. Could you please explain that?

**Comment by alkisg** [comment, marked as answer]
I assume that `192.168.5.1` is the LTSP server, while `192.168.67.1` and `192.168.67.2` are the addresses that LTSP uses to detect **the route** to the LTSP server, to prefer the possible internal subnet instead of external ones.

I think the problem isn't in all these IPs, but in the netplan initramfs hooks, which don't manage to set up an IP inside the initramfs for some reason. We'll need to check this issue online.

**Comment by keerthi-varuman** [comment]
**Managed to fix** 

The problem is ltsp initrd.img not updated properly.
So I deleted the old initrd.img and re-generated new one (**ltsp initrd**) after that all works fine.
